src/analysis/locally_convex/basic.lean

```diff
@@ -127,6 +156,16 @@ end has_scalar
 section add_comm_monoid
 variables [add_comm_monoid E] [module 𝕜 E] {s t u v A B : set E}
 
+lemma absorbs.add {s1 s2 t1 t2 : set E} (h1 : absorbs 𝕜 s1 t1) (h2 : absorbs 𝕜 s2 t2) :
+  absorbs 𝕜 (s1 + s2) (t1 + t2) :=
```

Comment:
Is there a version with `absorbs 𝕜 s (t₁ + t₂)` as the solution? Also, could you pull out `s₁ s₂ t₁ t₂` to the `variables`?